packages/apollo-server-testing/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "apollo-server-testing",
```

Comment:
This `apollo-server-testing` package should also be added to the monorepo's root [`package.json`](https://github.com/apollographql/apollo-server/blob/master/package.json)'s `dependencies and also to the [`tsconfig.build.json`](https://github.com/apollographql/apollo-server/blob/master/tsconfig.build.json) so it's picked up by the overall `tsc` build.

Also... sloppy as this is, I think it needs to be integrated into the Jest module mapper RegExp [here](https://github.com/apollographql/apollo-server/blob/1553f7d43c278e3f137e7d16aeac26dc254681b5/jest.config.base.js#L24).

Reply:
What do I need to add to the module mapper?

Reply:
Ah, nevermind.  It fits into the same match as the integration packages.  Let's hope that nobody ever releases an HTTP server named `testing`. :trollface: